src/quipper/perf_buildid.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 The Chromium OS Authors. All rights reserved.
```

Comment:
2016 is not right.

Reply:
Good catch, will update those and overwrite this branch with both this change and an addition of authors to get past the CLA failures.